.travis.yml

```diff
@@ -1,8 +1,3 @@
 language: node_js
 node_js:
-    - 4
-script: gulp
-before_script:
-    - npm install -g istanbul
```

Comment:
You should consider using just 'npm install' to install all depencies before_script as istanbul is not the only required dependency